src/uneditable-section/section-item.tsx

```diff
@@ -161,6 +168,11 @@ export const UneditableSectionItem = ({
                 aria-busy={maskLoadingState === "loading"}
                 aria-live="polite"
                 type="button"
+                title={StringHelper.getMaskedDescription(
```

Comment:
do not use `title` attribute. use `aria-label` instead to describe the masked input.  and `aria-label` should only be set on the masked children. ensure that this does not override the accessible label for the fail and loading state